stripe/http_client.py

```diff
@@ -494,21 +494,24 @@ def _handle_request_error(self, e):
                 "https://twitter.com/stripestatus, or let us know at "
                 "support@stripe.com."
             )
+            should_retry = True
         elif e.args[0] in [pycurl.E_SSL_CACERT, pycurl.E_SSL_PEER_CERTIFICATE]:
             msg = (
                 "Could not verify Stripe's SSL certificate.  Please make "
                 "sure that your network is not intercepting certificates.  "
                 "If this problem persists, let us know at "
                 "support@stripe.com."
             )
+            should_retry = False
```

Comment:
I don't think we should retry this one — if the SSL certificate couldn't be verified on the first request, it's not likely to be verified on the next either.

Reply:
We're not retrying, `should_retry` is set to `False` here, not `True`.